cime_config/testdefs/testlist_clm.xml

```diff
@@ -2513,15 +2514,46 @@
       <option name="comment"  >Transient production future scenario SSP3-7.0 case with isotopes with a december 2050 start</option>
     </options>
   </test>
-  <test name="SMS_D_Ld5" grid="f09_g17" compset="ISSP126Clm50BgcCrop" testmods="clm/datm_ssp126_anom_forc">
+  
+  <test name="SMS_D_Ld5" grid="f09_g17" compset="ISSP126Clm50BgcCrop" testmods="clm/default">
```

Comment:
We actually shouldn't have a testmod for this. We want to have this handled based on the compset, so you shouldn't need a testmod for it.

Reply:
Or is this something particular about the ssp126 case? You don't do this with the other SSP's below...

Reply:
The change I made was to _remove_ the ssp126-specific testmod. It's the only SSP that had one.